src/twisted/conch/ssh/keys.py

```diff
@@ -863,21 +987,42 @@ def type(self):
         elif isinstance(
                 self._keyObject, (dsa.DSAPublicKey, dsa.DSAPrivateKey)):
             return 'DSA'
+        elif isinstance(
+                self._keyObject, (ec.EllipticCurvePublicKey, ec.EllipticCurvePrivateKey)):
+            return 'EC'
         else:
             raise RuntimeError(
                 'unknown type of object: %r' % (self._keyObject,))
 
 
+    def getECKeyName(self):
```

Comment:
I don't understand the purpose of this method; I think it should probably be removed.  Nothing external calls it - and nothing should.  The purpose of `Key` is to present a uniform interface for the operations supported by all key types.